docs/CLI.md

```diff
@@ -13,8 +13,8 @@ You need the latest version of NPM installed. There are multiple ways to downloa
 Clone the GitHub repository and cd into that directory.
 
 ```bash
-git clone https://github.com/GSA/open-product-accessibility-template.git
-cd open-product-accessibility-template
+git clone https://github.com/GSA/OpenACR.git
```

Comment:
@mgifford let's go with lowercase here and the next line.

Reply:
I've updated these two lines.